pkg/cmd/scan.go

```diff
@@ -122,6 +124,9 @@ func NewScanCmd() *cobra.Command {
 }
 
 func scanRun(opts *ScanOptions) error {
+	selectedOutput := output.GetOutput(opts.Output)
+	globaloutput.ChangePrinter(selectedOutput.GetInfoPrinter())
```

Comment:
It does not really makes sense to set the global printer there, it could be done within the output pkg IMHO

Reply:
We still need the selectedOutput to write the anlysis I'm not sure how you see this ?